Assets/Scripts/RendererFeatures/Blit.cs

```diff
@@ -37,20 +36,17 @@ public override void Create()
 
         public override void AddRenderPasses(ScriptableRenderer renderer, ref RenderingData renderingData)
         {
-            if (!settings.onlyMainCamera || renderingData.cameraData.camera.tag.Equals("MainCamera")) // added
-            {
-                var src = renderer.cameraColorTarget;
-                var dest = (settings.destination == Target.Color) ? RenderTargetHandle.CameraTarget : m_RenderTextureHandle;
-
-                if (settings.blitMaterial == null)
-                {
-                    Debug.LogWarningFormat("Missing Blit Material. {0} blit pass will not execute. Check for missing reference in the assigned renderer.", GetType().Name);
-                    return;
-                }
+            var src = renderer.cameraColorTarget;
+            var dest = (settings.destination == Target.Color) ? RenderTargetHandle.CameraTarget : m_RenderTextureHandle;
```

Comment:
what does this mean?

Reply:
The only thing I changed in this script was getting rid of modifications I made previously. Now it is purely the script from the demo project. I had made a change to allow you to only target the main camera with a blit, but that was before I realized there was a built-in way to do it. If you look at this file's diff, you'll see that the lines you commented on were already in the script, I just unindented them.

Reply:
ah ok got it